_extras/snippets_library/UCL_sge/14/depend_demo.snip

```diff
@@ -1,47 +1,59 @@
-To demonstrate, let's load the `abinit` module and then use the `module list` command to show
-which modules we currently have loaded in our environment. ([Abinit](https://www.abinit.org/) is an open source 
-materials science modelling software package.)
+To demonstrate, let's load the `ansys` module and then use the `module list` command to show
+which modules we currently have loaded in our environment. ([ANSYS](https://www.ansys.com/en-gb) is
+an engineering simulation product.)
 
 ```
-{{ site.host_prompt }} module load abinit
-{{ site.host_prompt }} module list
+{{ site.host_prompt }} module load ansys
 ```
 {: .bash}
-```
-Currently Loaded Modulefiles:
-  1) anaconda/python3                  5) intel-compilers-17/17.0.2.174     9) netcdf/4.4.1
-  2) mpt/2.16                          6) intel-cmkl-17/17.0.2.174         10) abinit/8.2.3-intel17-mpt214
-  3) intel-cc-17/17.0.2.174            7) gcc/6.2.0
-  4) intel-fc-17/17.0.2.174            8) fftw-3.3.5-intel-17.0.2-dxt2dzn
+ansys/2019.r3(73):ERROR:151: Module 'ansys/2019.r3' depends on one of the module(s) 'giflib/5.1.1'
+ansys/2019.r3(73):ERROR:102: Tcl command execution failed: prereq   giflib/5.1.1
 ```
 {: .output}
+This shows that the default `ansys` module will not run because it first needs `giflib/5.1.1` to be loaded.
```

Comment:
leave a blank line after the flags, though it may render fine, it will be better for other automation process (like tokenization for translation).

Reply:
I've left some more blank lines. Please let me know if that doesn't fix it.

Reply:
If you merge master to your branch, you can view the rendered webpage in circleCI

Reply:
Not master, gh-pages, damnit!